client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -94,6 +99,15 @@ const PluginsBrowserList = ( {
 				</div>
 				<div className="plugins-browser-list__subtitle">{ subtitle }</div>
 			</div>
+			{ listName === 'featured' && isEnabled( 'marketplace-spotlight' ) && (
+				<Spotlight
+					taglineText={ translate( 'Drive more traffic with Yoast SEO' ) }
+					titleText={ translate( 'Under the Spotlight' ) }
+					ctaText={ translate( 'View Details' ) }
+					illustrationSrc={ yoastPlugin?.icon ?? '' }
```

Comment:
I don't think we can rely on [this prop](https://github.com/Automattic/wp-calypso/pull/61185/files/b56e3b724c64b3ad079b3543349b3628eb763351#diff-56bb43ea9c8036c7ad5823056d3b656e8b9482ee4285b374628c5cd50ffccb74R33) to always contain yoast.

Do you think we could just hardcode the image url here?

Reply:
```suggestion
					illustrationSrc={ spotlightPlugin?.icon ?? '' }
```

If we use `spotlightPlugin` we should be good. Either we have added the icon in the backend or not. It depends though on the illustration. @SaxonF would that spotlight always use the plugin icon? If not, let's hardcode the image url in Calypso.

Reply:
✅